demo/date-picker-presentation-demos.html

```diff
@@ -58,6 +58,22 @@ <h3>Date picker with week numbers</h3>
       </template>
     </vaadin-demo-snippet>
 
+
+    <h3>Date picker with reference Date</h3>
+    <p> Testing reference date. (Stting up reference date to: 1919-11-31, so giving the following as input dd/MM/70(year) will be year 1870, and dd/MM/68(year) will be year 1968, where dd is days, MM is months)</p>
```

Comment:
I think that maybe it could be better if the wording focuses mostly on the interpretation of the short year. What do you think?

```suggestion
    <p>Dates with 2-digit years will be interpreted according to the reference date, 1919-11-31 in this example. Therefore 70 will be interpreted as 1870, while 68 will be interpreted as 1968.</p>
```

Reply:
Sure, let's do this! Your wording makes it more clear to me as well, also just wanted to help that as I remember day and month are needed too to parse it properly.